pyqir-generator/src/python.rs

```diff
@@ -30,11 +33,16 @@ impl PyQIR {
         }
     }
 
+    #[allow(clippy::needless_pass_by_value)]
+    #[allow(clippy::unnecessary_wraps)]
+    #[allow(clippy::unused_self)]
     fn add_measurement(&mut self, qubit: String, target: String) -> PyResult<()> {
         log::info!("measure {} => {}", qubit, target);
         Ok(())
     }
 
+    #[allow(clippy::needless_pass_by_value)]
```

Comment:
Is Clippy really warning here? Doesn't `Controlled::new` consume the strings?

Reply:
Yes, it wants `&str` as that is all the logging needs but I wanted all of the python/rust interfaces to have owned objects.

Reply:
I just tried it. Removing `#[allow(clippy::needless_pass_by_value)]` from `PyQIR::cx`, `PyQIR::cz`, etc. doesn't produce any lints. Are you thinking of `PyQIR::add_measurement`?

Reply:
You are right. Thank you!